src/Command/InfectionCommand.php

```diff
@@ -104,7 +104,7 @@ protected function configure(): void
                 'j',
                 InputOption::VALUE_REQUIRED,
                 'Threads count',
-                1
+                '1'
```

Comment:
The input hints on this method changed in one of the later version of symfony to no longer allow ints.